pkg/cmd/roachprod/cli/flags.go

```diff
@@ -200,6 +201,10 @@ func initListCmdFlags(listCmd *cobra.Command) {
 		"cost", "c", os.Getenv("ROACHPROD_COST_ESTIMATES") == "true",
 		"Show cost estimates",
 	)
+	listCmd.Flags().BoolVarP(&listSparkline,
+		"spark", "s", os.Getenv("ROACHPROD_SPARK_LINE") == "true",
+		"Show sparkline",
```

Comment:
nit: This flag description could be fleshed out a bit. Something mentioning: `Show sparkline graph of txn_commits over last 5 minutes. Currently only supported on iterm`?

Reply:
Sure thing, I'll add some text to describe the flag.